entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+export GOOGLE_APPLICATION_CREDENTIALS="/usr/src/app/gcloud_key.json"
+gcloud config set project k8s-jkns-gke-soak
```

Comment:
I would suggest parameterizing this as an env var with a default, e.g.
```suggestion
GCP_PROJECT="${GCP_PROJECT:-k8s-jkns-gke-soak}"
gcloud config set project "${GCP_PROJECT}"
```

Reply:
done